ibm_mq/datadog_checks/ibm_mq/connection.py

```diff
@@ -56,3 +55,13 @@ def get_ssl_connection(config):
     queue_manager.connect_with_options(config.queue_manager_name, cd, sco)
 
     return queue_manager
+
+
+def _get_cd(config):
```

Comment:
I know the pyqmi docs/API uses 'CD' too, but perhaps we can use this if this is what `cd` means…?

```suggestion
def _get_channel_definition(config):
```

And then use `channel_definition = ...` as a variable name?